main.go

```diff
@@ -183,7 +183,12 @@ func visit(url *url.URL) {
 		log.Fatalf("unable to create request: %v", err)
 	}
 	for _, h := range httpHeaders {
-		req.Header.Add(headerKeyValue(h))
+		k, v := headerKeyValue(h)
+		if strings.EqualFold(k, "host") {
```

Comment:
Answered my own question: https://github.com/davecheney/httpstat/pull/71/files#r80565114

Sorry for the noise.